### PR TITLE
support for connection_name on multi-source replication

### PR DESCRIPTION
**Pull request by diegobill**
(https://mariadb.com/kb/en/library/multi-source-replication/)

##### SUMMARY
support for connection_name on multi-source replication (https://mariadb.com/kb/en/library/multi-source-replication/)

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
module mysql mysql_replication

##### ANSIBLE VERSION
```
ansible 2.8.0.dev0
  config file = None
  configured module search path = ['/home/felix/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/felix/workspace/tapweb.cgi/.env/lib/python3.5/site-packages/ansible
  executable location = /home/felix/workspace/tapweb.cgi/.env/bin/ansible
  python version = 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609]
```

### CODE REVIEW

File: lib/ansible/modules/database/mysql/mysql_replication.py [hunk 1]
```diff
@@ -88,6 +88,9 @@
         description:
             - does the host uses GTID based replication or not
         version_added: "2.0"
+    connection_name:
```
Comment (bcoca) [inline lib/ansible/modules/database/mysql/mysql_replication.py:91]: missing version_added

**Review by bcoca — CHANGES_REQUESTED** [review]

**Comment by CyberBLN** [comment]
@diegobill Can you please rebase your branch to get it merged into official repo? Thank you :)

**Comment by Andersson007** [comment]
@diegobill  , hello, thank you for the pr
I've recently added integration tests for mariadb from scratch ```test/integration/targets/mariadb_replication/tasks/```.
Now it is possible to test mysql_replication module.
So, could you please
1. add tests for this pr by separate file and including it to test/integration/targets/mariadb_replication/tasks/main.yml
2. An example of using this parameter to Example block

After that i review it.
(Would be cool to get a feedback ASAP, i can wait a couple of days - no more :) , then, if you're busy, i could rewrite it, no problem)
Thank you!

**Comment by Andersson007** [comment]
@gundalow , please, close this via https://github.com/ansible/ansible/pull/63229

**Comment by gundalow** [comment]
resolved_by_pr: 63229

**Pull request closed without merging**